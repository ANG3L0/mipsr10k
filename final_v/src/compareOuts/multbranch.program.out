Chronologic VCS simulator copyright 1991-2011
Contains Synopsys proprietary information.
Compiler version E-2011.03_Full64; Runtime version E-2011.03_Full64;  Dec 10 00:51 2012
@@
@@
@@                     0  Asserting System reset......
@@
@@                     0 : System STILL at reset, can't show anything
@@
@@
@@                    10 : System STILL at reset, can't show anything
@@
@@
@@                    20 : System STILL at reset, can't show anything
@@
@@@ Unified Memory contents hex on left, decimal on right: 
@@@
@@@ mem[    0] = 205f000a203f0001 : 2332583175491878913
@@@ mem[    8] = 209f0008207f0002 : 2350597565415620610
@@@ mem[   16] = 4083052640830404 : 4648564902225708036
@@@ mem[   24] = 4c63040340410522 : 5504247581586294050
@@@ mem[   32] = 40410409f45ffffd : 4629986332714729469
@@@ mem[   40] = 0000000000000555 : 1365
@@@
@@@
@@                  1112 : System halted
@@
@@@ System halted on HALT instruction
@@@
@@
@@  110 cycles / 52 instrs = 2.115385 CPI
@@
@@  3270.00 ns total time to execute
@@

$finish called from file "testbench/pipeline_test.v", line 817.
$finish at simulation time                 1112
           V C S   S i m u l a t i o n   R e p o r t 
Time: 1112
CPU Time:      0.300 seconds;       Data structure size:   0.3Mb
Mon Dec 10 00:51:22 2012
