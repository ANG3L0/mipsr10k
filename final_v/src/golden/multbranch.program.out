Chronologic VCS simulator copyright 1991-2012
Contains Synopsys proprietary information.
Compiler version G-2012.09_Full64; Runtime version G-2012.09_Full64;  Dec 10 00:19 2012
@@
@@
@@                     0  Asserting System reset......
@@
@@                     0 : System STILL at reset, can't show anything
@@
@@
@@                   100 : System STILL at reset, can't show anything
@@
@@
@@                   200 : System STILL at reset, can't show anything
@@
@@
@@                   300 : System STILL at reset, can't show anything
@@
@@                   360  Deasserting System reset......
@@
@@
@@@ Unified Memory contents hex on left, decimal on right: 
@@@
@@@ mem[    0] = 205f000a203f0001 : 2332583175491878913
@@@ mem[    8] = 209f0008207f0002 : 2350597565415620610
@@@ mem[   16] = 4083052640830404 : 4648564902225708036
@@@ mem[   24] = 4c63040340410522 : 5504247581586294050
@@@ mem[   32] = 40410409f45ffffd : 4629986332714729469
@@@ mem[   40] = 0000000000000555 : 1365
@@@
@@@
@@                 20920 : System halted
@@
@@@ System halted on HALT instruction
@@@
@@
@@  206 cycles / 37 instrs = 5.567568 CPI
@@
@@  6150.00 ns total time to execute
@@

$finish called from file "testbench/testbench.v", line 290.
$finish at simulation time                21920
           V C S   S i m u l a t i o n   R e p o r t 
Time: 2192000 ps
CPU Time:      0.440 seconds;       Data structure size:   0.1Mb
Mon Dec 10 00:19:34 2012
